fbw-a32nx/src/base/flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml

```diff
@@ -4734,17 +4734,17 @@
                 <PART_ID>MASTERWARN_R</PART_ID>
                 <LEFT_SINGLE_CODE>
                     0 (&gt;L:A32NX_MASTER_WARNING)
-                    1 (&gt;L:PUSH_AUTOPILOT_MASTERAWARN_L)
+                    1 (&gt;L:A32NX_FWS_MW_CANCEL_ON_FO)
                 </LEFT_SINGLE_CODE>
                 <LEFT_LEAVE_CODE>
-                    0 (&gt;L:PUSH_AUTOPILOT_MASTERAWARN_L)
+                    0 (&gt;L:A32NX_FWS_MW_CANCEL_ON_FO)
                 </LEFT_LEAVE_CODE>
                 <!-- 202XP-A via CB 27LP -->
                 <SEQ1_POWERED>(L:A32NX_ELEC_AC_2_BUS_IS_POWERED)</SEQ1_POWERED>
-                <SEQ1_CODE>(L:Generic_Master_Warning_Active) (L:A32NX_OVHD_INTLT_ANN) 0 == or</SEQ1_CODE>
+                <SEQ1_CODE>(L:Generic_Master_Warning_Active) (L:A32NX_FWC_2_NORMAL) 1 == and (L:A32NX_OVHD_INTLT_ANN) 0 == or</SEQ1_CODE>
                 <!-- 801XP-C via CB 29LP -->
                 <SEQ2_POWERED>(L:A32NX_ELEC_AC_ESS_SHED_BUS_IS_POWERED)</SEQ2_POWERED>
-                <SEQ2_CODE>(L:Generic_Master_Warning_Active) (L:A32NX_OVHD_INTLT_ANN) 0 == or</SEQ2_CODE>
+                <SEQ2_CODE>(L:Generic_Master_Warning_Active) (L:A32NX_FWC_1_NORMAL) 1 == and (L:A32NX_OVHD_INTLT_ANN) 0 == or</SEQ2_CODE>
```

Comment:
I think the Lvars are `L:A32NX_FWS_FWC_1_NORMAL` and `L:A32NX_FWS_FWC_2_NORMAL`.